packages/playground/Samples/simple.tsx

```diff
@@ -3,19 +3,46 @@
  * Licensed under the MIT License.
  * @format
  */
-import React from 'react';
-import {AppRegistry, View} from 'react-native';
+import React, {useState} from 'react';
```

Comment:
I don't think we want to change the simple page here, if we wanted to add tests or anything it should go in rn-tester

Reply:
Okay - I'll just revert this back